packages/flutter_tools/templates/plugin/pubspec.yaml.tmpl

```diff
@@ -3,6 +3,9 @@ description: {{description}}
 
 flutter:
   plugin:
+    version: 1
```

Comment:
This was the plugin interface version that the plugin implemented, not the version of the plugin itself.

Removed to avoid confusion. When we need to introduce it, we'll have to pick a better name.